web/apps/apps.go

```diff
@@ -228,17 +228,8 @@ func logsHandler(appType consts.AppType) echo.HandlerFunc {
 	return func(c echo.Context) error {
 		inst := middlewares.GetInstance(c)
 
-		var slug string
-		err := middlewares.AllowMaximal(c)
-		if err == nil {
-			// If logs are sent by the flagship app, get the slug from the
-			// request params.
-			slug = c.Param("slug")
-			_, err := app.GetBySlug(inst, slug, appType)
-			if err != nil {
-				return wrapAppsError(err)
-			}
-		} else {
+		slug := c.Param("slug")
+		if err := middlewares.AllowMaximal(c); err != nil {
 			// If logs are not sent by the flagship app, check that it's sent by
```

Comment:
This seems unnecessary if we only allow flagship apps to send logs.

Reply:
Yes, but I would like to keep it as it can be useful for debugging

Reply:
OK. Maybe update the comment then, since we won't get into this block if logs are not sent by the flagship app.

Reply:
I don't understand: we go in this block if logs are not sent by the flagship app, it's the purpose of this block

Reply:
Aaah I'm sorry, I misread the condition… This is totally fine!